docs/DeveloperGuide.md

```diff
@@ -33,6 +140,55 @@
 
 * *glossary item* - Definition
 
+## Appendix: Requirements
+
+### Product scope
+
+BankWithUs is a powerful and user-friendly CLI-based banking system 
+that provides users with an easy and quick way to manage their financial 
+transactions and budget accordingly. With its wide range of features and 
+easy-to-use command syntax, BankWithUs makes it simple for users to manage their finances on the go.
+
+
+### Target user profile
+
+The target audience for the command line-based Banking System is individuals who value 
+efficiency and simplicity in their financial management. This group includes students, 
+freelancers, and small business owners who are looking for a fast and straightforward way to manage their finances.
+
+
+### Value proposition
+
+Fast typers may benefit more from the CLI 
+version of the banking system as it allows for 
+quick navigation and interaction with the system using keyboard shortcuts. 
+This can lead to a faster and more efficient workflow compared to using a graphical user interface (GUI), 
+which may require more clicks and navigation. Additionally, fast typers may prefer the simplicity and 
+minimalism of a CLI interface, as it provides only the essential information and features needed to manage their finances.
+
+
+### User Stories
+
+| Version | As a ... | I want to ...                            | So that I can ...                                                                       |
+|---------|----------|------------------------------------------|-----------------------------------------------------------------------------------------|
+| v1.0    | user     | Add deposits                             | so that I can check my financial status.                                                |
+| v1.0    | user     | Withdraw money                           | so that i can see the effect of my spending on balance                                  |
+| v1.0    | user     | view account                             | so that i can keep track of my minimum balance                                          |
+| v2.0    | user     | add a withdrawal limit                   | so that i can be forced to maintain a frugal habit                                      |
+| v2.0    | user     | add a savings goal                       | so that i can keep track of my spendings in tandem with my savings goal                 |
+| v2.0    | user     | add a new account                        | so that i can categorise my accounts and manage financial better                        |
+| v2.0    | user     | switch account                           | so that i can navigate between the various accounts i have and keep track of finacnials |
+| v2.0    | user     | list all the command that can be used    | so that i can have a basic overview of the apps functionalities                         |
+
+
+### Non-Functional Requirements
+
+{Give non-functional requirements}
+
+### Glossary
+
+* *glossary item* - Definition
```

Comment:
It should be deleted, or replaced instead? It seems repetitive, since I saw another line in the down below.